docs/topics/mta-cli-run-single-app.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="mta-cli-run-single-app_{context}"]
+= Running the {ProductShortName} {CLINameTitle} against an application
+
+You can run the {ProductFullName} {CLINameTitle} against an application.
+
+.Procedure
+
+. Open a terminal and navigate to the `<{ProductShortName}_HOME>/` directory.
+
+. Run the `{mta-cli}` script, or `{mta-cli}.exe` for Windows, and specify the appropriate arguments:
+
++
+[source,terminal,subs="attributes+"]
+----
+$ ./{mta-cli} analyze --input <path_to_input> \
+    --output <path_to_output> --source <source_name> --target <target_source> \
+----
++
+* `--input`: The application to be evaluated.
+* `--output`: The output directory for the generated reports.
+* `--source`: The source technology for the application migration. For example, `weblogic`.
+* `--target`: The target technology for the application migration. For example, `eap8`.
+
+. Access the report.
+
+[id="command-examples_{context}"]
+== {ProductShortName} command examples
+
+[discrete]
+=== Running {ProductShortName} on an application archive
+
+The following command analyzes the example EAR archive named `jee-example-app-1.0.0.ear` for migrating from JBoss EAP 5 to JBoss EAP 7:
+
+[source,terminal,subs="attributes+"]
+----
+$ <{ProductShortName}_HOME>/{mta-cli} analyze \
+    --input <path_to_jee-example-app-1.0.0.ear> \
+    --output <path_to_report_output> --source eap5 --target eap7 \
+----
+[]
+
+[discrete]
+=== Running {ProductShortName} on source code
+
+The following command analyzes the example source code named `seam-booking-5.2` for migrating to JBoss EAP 8.
```

Comment:
This sentence doesn't make sense, source code doesn't have a name, the application does. It should read more like "The following command analyzes the source code of an example application called X". The name of the application itself is a bad example as well, since Seam framework is not supported in EAP 8, so this could be misleading. Replace with "customer-management". On top of that, versions would never be on an app name (which would be aligned with the source code/project directory), but only on the binaries produced out of it, like  "jee-example-app-1.0.0.ear"